Dockerfile.moc-xdmod

```diff
@@ -25,7 +25,7 @@ RUN yum makecache && \
     libreoffice \
     mariadb-server mariadb cronie logrotate \
     perl-Image-ExifTool php-mbstring php-pecl-apcu jq \
-    chromium-headless librsvg2-tools
+    chromium-headless librsvg2-toolsm 
```

Comment:
I think this is a typo: there is a package `librsvg2-tools`, but there is no package `librsvg2-toolsm`.